streaming/src/main/java/org/apache/carbondata/streaming/CarbonStreamRecordWriter.java

```diff
@@ -140,7 +140,9 @@ private void initialize(TaskAttemptContext job) throws IOException {
 
     segmentDir = CarbonTablePath.getSegmentPath(
         carbonTable.getAbsoluteTableIdentifier().getTablePath(), segmentId);
-    fileName = CarbonTablePath.getCarbonDataFileName(0, taskNo + "", 0, 0, "0", segmentId);
+    fileName = CarbonTablePath.getCarbonDataFileName(
```

Comment:
why always snappy for streaming segment ? before I think it was supporting configured compressor

Reply:
There is testcase failing if set to ZSTD. Need another PR for it

Reply:
ok, please add a TODO